pyart/aux_io/arm_vpt.py

```diff
@@ -0,0 +1,351 @@
+
+    if 'target_scan_rate' in ncvars:
+        target_scan_rate = cfradial._ncvar_to_dict(ncvars['target_scan_rate'])
+    else:
+        target_scan_rate = None
+    if 'rays_are_indexed' in ncvars:
+        rays_are_indexed = cfradial._ncvar_to_dict(ncvars['rays_are_indexed'])
+    else:
+        rays_are_indexed = None
+    if 'ray_angle_res' in ncvars:
+        ray_angle_res = cfradial._ncvar_to_dict(ncvars['ray_angle_res'])
+    else:
+        ray_angle_res = None
+
+    # first sweep mode determines scan_type
+    # this module is specific to vertically-pointing data
+    mode = 'vertical_pointing'
+    scan_type = 'vpt'
+
+    # 4.8 Sensor pointing variables -> create attribute dictionaries
+    # this section also required some changes since the initial NetCDF did not
+    # contain any sensor pointing variables
+    azimuth = {
+        'data': 0.0 * np.ones(ncvars['time'].size, dtype=np.float32),
+        'long_name': 'Azimuth angle relative to true north',
+        'standard_name': 'beam_azimuth_angle',
+        'units': 'degrees',
+        'axis': 'radial_azimuth_angle',
+        'comment': 'Azimuth of antenna relative to true north',
+    }
+    elevation = {
+        'data': 90.0 * np.ones(ncvars['time'].size, dtype=np.float32),
+        'long_name': 'Elevation angle from horizontal plane',
+        'standard_name': 'beam_elevation_angle',
+        'units': 'degrees',
+        'axis': 'radial_elevation_coordinate',
+        'comment': 'Elevation of antenna relative to the horizontal plane',
+    }
+
+    #azimuth = _ncvar_to_dict(ncvars['azimuth'])
+    #elevation = _ncvar_to_dict(ncvars['elevation'])
+
+    if 'scan_rate' in ncvars:
+        scan_rate = cfradial._ncvar_to_dict(ncvars['scan_rate'])
+    else:
+        scan_rate = None
+
+    if 'antenna_transition' in ncvars:
+        antenna_transition = cfradial._ncvar_to_dict(
+            ncvars['antenna_transition'])
+    else:
+        antenna_transition = None
+
+    # 4.9 Moving platform geo-reference variables
+    # Aircraft specific varaibles
+    if 'rotation' in ncvars:
+        rotation = _ncvar_to_dict(ncvars['rotation'])
+    else:
+        rotation = None
+    if 'tilt' in ncvars:
+        tilt = _ncvar_to_dict(ncvars['tilt'])
+    else:
+        tilt = None
+    if 'roll' in ncvars:
+        roll = _ncvar_to_dict(ncvars['roll'])
+    else:
+        roll = None
+    if 'drift' in ncvars:
+        drift = _ncvar_to_dict(ncvars['drift'])
+    else:
+        drift = None
+    if 'heading' in ncvars:
+        heading = _ncvar_to_dict(ncvars['heading'])
+    else:
+        heading = None
+    if 'pitch' in ncvars:
+        pitch = _ncvar_to_dict(ncvars['pitch'])
+    else:
+        pitch = None
+    if 'georefs_applied' in ncvars:
+        georefs_applied = _ncvar_to_dict(ncvars['georefs_applied'])
+    else:
+        georefs_applied = None
+
+    # 4.10 Moments field data variables -> field attribute dictionary
+    if 'ray_n_gates' in ncvars:
```

Comment:
This if clause can also be removed, ARM's zenith radar should maintain a consistent number of gates in each ray.  The else clause should always apply.